### PR TITLE
Add service tests.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

File: service_tests/test_service.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+X1#!/usr/bin/env python
```
Comment (johngian) [inline service_tests/test_service.py]: Please fix typo here.

File: service_tests/test_service.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import os
+import time
+from urlparse import urljoin
+
+import requests
```
Comment (johngian) [inline service_tests/test_service.py]: Please swap lines to be alphabetically ordered.

File: service_tests/test_service.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+
+BASE_URL = os.environ.get('BASE_URL', 'https://snippets.mozilla.com')
+
+
+@pytest.fixture(scope='module')
+def snippets_request():
+    url = urljoin(
+        BASE_URL,
+        '/4/Firefox/32.0.3/20140923175406/Darwin_Universal-gcc3/test-bot-{time}/release/Darwin%2013.3.0/default/default/')
```
Comment (johngian) [inline service_tests/test_service.py]: I think that this would be better if you define it as a var.

**Comment by johngian** [comment]
lgtm r+

**Comment by glogiotatidis** [comment]
Thanks! :cactus:

**Pull request merged**